src/mantine-core/src/components/Group/Group.tsx

```diff
@@ -40,7 +40,7 @@ export const Group = forwardRef<HTMLDivElement, GroupProps>(
     }: GroupProps,
     ref
   ) => {
-    const count = Children.count(children);
+    const filteredChildren = (Children.toArray(children) as React.ReactElement[]).filter(Boolean);
```

Comment:
Please create a function that filters falsy children to `@mantine/core` utils folder. Call it `filter-falsy-children`.

Reply:
Okay I added that function